docs/source/performance/caching.md

```diff
@@ -292,6 +290,46 @@ query GetReaderBookTitle {
 }
 ```
 
+## Using with federation
+
+When using [Apollo Federation](https://www.apollographql.com/docs/federation), the `@cacheControl` directive and `CacheControlScope` enum may be defined in a subgraph's schema. An Apollo Server-based subgraph will calculate and set the cache hint for the response that it sends to the gateway as it would for a non-federated Apollo Server sending a response to a client. The gateway will then calculate the cache hint for the overall response based on the most restrictive settings among all of the responses received from the subgraphs involved in query plan execution.
```

Comment:
We should merge this PR once the appropriate federation and gateway PRs are released, and we should include the minimum versions of federation and gateway required to support this.

Reply:
For the minimum versions, do you think they should be included directly in the AS3 docs or in a new section in the federation docs?

Reply:
I think it would be great to have new docs on the federation site (or at least a link to here) — are you interested in writing them? I think we should also include here the minimum versions.